File: diancanba/diancanba-Restaurant/src/main/java/com/jarvis/diancanba/restaurant/service/DaybillitemService.java
package com.jarvis.diancanba.restaurant.service;

import com.jarvis.diancanba.base.entity.PageResult;
import com.jarvis.diancanba.base.pojo.DcbDaybillitem;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface DaybillitemService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<DcbDaybillitem> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(DcbDaybillitem daybillitem);
	
	
	/**
	 * 修改
	 */
	public void update(DcbDaybillitem daybillitem);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public DcbDaybillitem findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(DcbDaybillitem daybillitem, int pageNum, int pageSize);
	
}

File: diancanba/diancanba-Restaurant/src/main/webapp/js/service/daybillitemService.js
//服务层
app.service('daybillitemService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../daybillitem/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../daybillitem/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../daybillitem/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../daybillitem/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../daybillitem/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../daybillitem/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../daybillitem/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});
